app/lib/xpAutostore.js

```diff
@@ -0,0 +1,27 @@
+module.exports = Base => class extends Base {
+    constructor() {
+        super();
+    }
+
+    async storePlayersXpAutoStoreTrue() {
```

Comment:
You can just resolve this comment but I wanted to say I'm proud of you for this function.

I think this it's very clean concise, you can read it and understand what is doing